src/content/docs/apis/nerdgraph/get-started/introduction-new-relic-nerdgraph.mdx

```diff
@@ -150,38 +149,53 @@ Dashboards
 
     <tr>
       <td>
-Alerts      </td>
+Data partitions
+      </td>
 
       <td>
-[See all alert-related tutorials](/docs/alerts-applied-intelligence/new-relic-alerts/alerts-nerdgraph/nerdgraph-api-nrql-condition-alerts/)      </td>
+[Manage data partitions](/docs/apis/nerdgraph/examples/nerdgraph-data-partition-rules-tutorial)
+      </td>
     </tr>
 
- <tr>
+    <tr>
       <td>
-Applied Intelligence     </td>
+Data management</td>
```

Comment:
@paperclypse, this is outside the scope of this PR, but if you have time, you might consider adding some kind of brief lead-in to headless bullet lists. These lists almost melt into the previous table rows, making it harder for me to parse. For example, it could be a simple as this:

Data management tutorials:
  * Convert event data to metric data
  * Data retention
  * Drop data

Reply:
You could always file it as an issue @rhetoric101 and maybe someone will get to it some day :)